client-src/elements/form-definition.js

```diff
@@ -319,41 +287,36 @@ const IMPLSTATUS_ORIGINTRIAL = [
 
 const MOST_PREPARETOSHIP = [
   'tag_review', 'tag_review_status', 'non_oss_deps',
-  'webview_risks', 'anticipated_spec_changes', 'origin_trial_feedback_url',
-  'launch_bug_url', 'intent_to_ship_url', 'i2s_lgtms', 'comments', 'finch_url',
+  'webview_risks', 'anticipated_spec_changes', 'measurement',
+  'intent_to_ship_url', 'i2s_lgtms', 'finch_url',
 ];
 
-const ANY_SHIP = ['launch_bug_url', 'finch_url', 'comments'];
+const ANY_SHIP = ['launch_bug_url', 'finch_url'];
 
 const EXISTING_PROTOTYPE = [
-  'owner', 'editors', 'cc_recipients', 'blink_components', 'motivation',
-  'explainer_links', 'spec_link', 'standard_maturity', 'api_spec', 'bug_url',
-  'launch_bug_url', 'intent_to_implement_url', 'comments',
+  'explainer_links', 'spec_link', 'standard_maturity', 'api_spec',
+  'intent_to_implement_url',
 ];
 
 const EXISTING_ORIGINTRIAL = [
   'experiment_goals', 'experiment_risks',
   'experiment_extension_reason', 'ongoing_constraints',
   'intent_to_experiment_url', 'intent_to_extend_experiment_url',
-  'i2e_lgtms', 'origin_trial_feedback_url', 'comments',
+  'i2e_lgtms', 'origin_trial_feedback_url',
 ];
 
-const PSA_IMPLEMENT = ['motivation', 'spec_link', 'standard_maturity', 'comments'];
+const PSA_IMPLEMENT = ['spec_link', 'standard_maturity'];
```

Comment:
@jrobbins Do we still want to have different fields for the same general stage type? For instance, this definition excludes the `intent_to_implement_url`, `explainer_links`, and `spec_link` from the developer-facing code changes feature type.

Reply:
Regardless, this file likely needs a greater cleanup and organization to make these definitions easier to ingest and edit

Reply:
> @jrobbins Do we still want to have different fields for the same general stage type? For instance, this definition excludes the `intent_to_implement_url`, `explainer_links`, and `spec_link` from the developer-facing code changes feature type.

Yes, I expect that the fields offered in each stage will still be determined by the feature type.  That could happen indirectly because stage types are determined by feature type.